src/types/line.js

```diff
@@ -88,6 +89,7 @@ export default class LineAnnotation extends Element {
 				y = min;
 				y2 = max;
 			}
+			options._horizontal = !scale.isHorizontal();
```

Comment:
lets have this kind of things directly on the element

Reply:
oki